src/common/execution-provider/execution-provider.service.ts

```diff
@@ -22,4 +27,10 @@ export class ExecutionProviderService {
     const { chainId } = await this.provider.getNetwork();
     return chainId;
   }
+
+  public async getLatestWithdrawals(): Promise<Array<{ validatorIndex: string }>> {
+    const provider = new ethers.JsonRpcProvider(this.configService.get('EL_RPC_URLS')[0]);
```

Comment:
With this implementation, the functionality of fallbacks and logging is lost. "SimpleFallbackJsonRpcBatchProvider" doesn't fit here? Maybe you can still use it and there is only a typing problem?